containerscan/commondatastructures.go

```diff
@@ -24,6 +24,24 @@ type CommonContainerVulnerabilityResult struct {
 	Vulnerability `json:",inline"`
 }
 
+type CommonContainerVulnerabilityResultV2 struct {
```

Comment:
[Requred]: Please create a directory name `v2` and place this struct there. I don't mind if you move the `v1` struct to a `v1` directory (although it means we will need to fix the imports in the code...)